menu/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from menu.models import Category, SubCategory
+
+class CategoryView(View):
+    def get(self, request):
+         try:
+            categories = Category.objects.prefetch_related('subcategory_set').all()
+
+            if categories.exists():   
+                categories = [{ 
+                 'categoryId'              :  category.id,
+                 'category'           :  category.name,
+                 'subCategories' : [
+                     sub_category.name
+                for sub_category in category.subcategory_set.all()]
+            } for category in categories]
+             
+            return JsonResponse({"categories" : categories}, status=200)
+            
+         except ValueError:
+             return JsonResponse({'message':'VALUE_ERROR'}, status=400)
```

Comment:
```suggestion
         except ValueError:
             return JsonResponse({'message':'VALUE_ERROR'}, status=400)
```
위의 ValueError케이스가 날 수 있는 경우가 무엇 일까요?

Reply:
발생할 수 없는 에러인것 같습니다. 불필요하다고 생각하여 수정하겠습니다!

Reply:
아마도 타입이 다른 에러가 들어가면 날것 같은데, 이렇게 익셉션 처리를 하던지 아니면  타입 검사를 해서 TypeError를 raise 하는게 더 좋을것 같습니다.